introduction.md

```diff
@@ -0,0 +1 @@
+Hello all, I am Jake Rushing from Dice.com and I am taking this course to better myself in git. I hope that you think I'm awesome! :D
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.